src/Config_Options.cpp

```diff
@@ -118,7 +117,9 @@ void Config_Options::config_parse(MapContainer &mappings) {
 			enum json_type type = json_object_get_type(value);
 
 			if (strcmp(key, "daemon") == 0 && type == json_type_boolean) {
-				_daemon = json_object_get_boolean(value);
+				if (!json_object_get_boolean(value)) {
+					throw vz::VZException("\"daemon\" option is not supported anymore.");
```

Comment:
Eine Exception wenn daemon noch gesetzt ist? Das heißt doch vzlogger wird abgebrochen?
Falls ja finde ich das nicht so schick, das gibt einen Haufen Anfragen mit alten Konfigurationen wenn wir das durchziehen. Ich denke da reicht eine Warnung und laufen lassen mit dem Standard foreground=false.

Reply:
exception is thrown if "deamon" is set to FALSE ('!json_object_get_boolean()`),
which never worked to begin with, as you pointed out,
and would, if at all, be mainly used for testing.
this will warn users trying to set daemon=false, and redirect them to the new proper options.

Reply:
Danke für die Erklärung, dann passt das ja.